docker-compose/docker-compose.yml

```diff
@@ -14,13 +14,15 @@ services:
         max-size: "10m"
         max-file: "10"
     volumes:
+      - /etc/nginx/cert:/etc/nginx/cert
       - type: volume
         source: eidaws_federator_log
         target: /var/log/eidaws
         volume:
           nocopy: false
     ports:
-      - "8080:8080"
+      - "80:8080"
```

Comment:
For testing purposes I'd rather set the ports to nonprivileged ports. On the machine actually deploying the stuff, I had a *secret* commit on top where the ports were adjusted to the privileged port values.

Reply:
this would then apply to 443 as well, which is currently also mapped 1:1 ?

Reply:
Exactly. Alternatively, provide a HTTPS specific *override* config (see e.g. https://docs.docker.com/compose/extends/). This way, the HTTPS related config is decoupled from the HTTP (*base*) config (regarding docker-compose this is IMO the cleaner approach).

Besides, it would be nice to decouple HTTP and HTTPS in the context of the `eidaws-federator` container image.

Reply:
ack. ll do with an additional production compose file to overwrite standard ports

Reply:
done